### PR TITLE
feat: package sets

### LINKED ISSUE #246: feat: 目的に応じたプラグインを追加する機能
## 問題点

必要なプラグインを目的から探して一括でインストールできるようになれば便利です。

See also: https://github.com/team-apm/apm/issues/405

## 提案する解決方法

思いついたパッケージのセットを下に書いておきます。これらを依存パッケージ扱いでまとめてインストールできる新しいパッケージを作ることでプログラムの変更なしに機能を追加できます。

```
scrapbox.io/aviutl による推奨環境
L-SMASH Works MrOjii版 + InputPipePlugin + かんたんMP4出力 + patch.aul

AviUtl効率化 パッケージセット
シークバー＋ + 最終フレーム自動調整 + フィルタドラッグ移動 + エディットボックス最適化 + アイテム内音声波形 + LuaJIT

音声編集 パッケージセット
VSTホストプラグイン＋α + 少し音確認再生 + アイテム内音声波形
```

## 補足説明

対応するプルリクエスト
- https://github.com/team-apm/apm-data/pull/245

### PR DESCRIPTION
**Pull request by mitosagi**
## Description

Add package sets to apm.

## Related Issue

- closes https://github.com/team-apm/apm/issues/405
- closes #246 

## How Has This Been Tested?

- It can be installed with the release version of apm.

## Checklist:

- [ ] ~Updated the modification date in `mod.xml`.~

### CODE REVIEW

File: v3/package-sets.json [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        "aviutl1.10",
+        "exedit0.92",
+        "MrOjii/LSMASHWorks",
+        "amate/InputPipePlugin",
+        "aoytsk/easymp4",
+        "ePi/patch"
+      ],
+      "pageURL": "https://scrapbox.io/aviutl/セットアップ",
+      "downloadURLs": [
+        "https://mitosagi.github.io/apm-data/package-sets-description.html"
```
Comment (hal-shu-sato) [inline v3/package-sets.json:20]:
```suggestion
        "https://team-apm.github.io/apm-data/package-sets-description.html"
```

File: v3/package-sets.json [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        "aoytsk/seekbarx",
+        "hebiiro/adjust",
+        "hebiiro/DragFilter",
+        "hebiiro/OptimizeEditBox",
+        "hebiiro/ShowWaveform",
+        "ePi/LuaJIT"
+      ],
+      "pageURL": "https://github.com/team-apm/apm-data",
+      "downloadURLs": [
+        "https://mitosagi.github.io/apm-data/package-sets-description.html"
```
Comment (hal-shu-sato) [inline v3/package-sets.json:41]:
```suggestion
        "https://team-apm.github.io/apm-data/package-sets-description.html"
```

File: v3/package-sets.json [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      "description": "AviUtlでより良く音声を扱うためのパッケージを追加できます。",
+      "developer": "apm",
+      "dependencies": [
+        "AiosCiao/VSThost4aviutl",
+        "rikky/smallPlay",
+        "hebiiro/ShowWaveform"
+      ],
+      "pageURL": "https://github.com/team-apm/apm-data",
+      "downloadURLs": [
+        "https://mitosagi.github.io/apm-data/package-sets-description.html"
```
Comment (hal-shu-sato) [inline v3/package-sets.json:59]:
```suggestion
        "https://team-apm.github.io/apm-data/package-sets-description.html"
```

**Review by hal-shu-sato — APPROVED** [review]
The packages are installed correctly.
Before merging, do you change these?

**Comment by mitosagi** [comment]
I have fixed it.

**Review by hal-shu-sato — APPROVED** [review]

**Pull request merged**